input/drivers/udev_input.c

```diff
@@ -757,6 +766,49 @@ static bool udev_pointer_is_off_window(const udev_input_t *udev)
 #endif
 }
 
+static int16_t udev_lightgun_aiming_state(udev_input_t *udev, unsigned idx, unsigned id )
+{
+   const int edge_detect = 32700;
+   struct video_viewport vp;
+   bool inside                 = false;
+   int16_t res_x               = 0;
+   int16_t res_y               = 0;
+   int16_t res_screen_x        = 0;
+   int16_t res_screen_y        = 0;
+
+   vp.x                        = 0;
+   vp.y                        = 0;
+   vp.width                    = 0;
+   vp.height                   = 0;
+   vp.full_width               = 0;
+   vp.full_height              = 0;
+
+   udev_input_mouse_t *mouse = udev_get_mouse(udev, port);
```

Comment:
Build error. Changed this to -

udev_input_mouse_t *mouse = udev_get_mousee(udev, idx);